packages/compartment-mapper/README.md

```diff
@@ -225,8 +225,12 @@ To overcome such obstacles, the compartment mapper will accept a non-standard
 `js` to the corresponding language name, one of `mjs` for ECMAScript modules,
 `cjs` for CommonJS modules, and `json` for JSON modules.
 All other language names are reserved and the defaults for files with the
-extensions `cjs`, `mjs`, and `json` default to the language of the same name
-unless overridden.
+extensions `cjs`, `mjs`, `json`, `text`, and `bytes` default to the language of
```

Comment:
.txt is conventional. Did you consider it?

Reply:
I guess folks can say .txt in package.json...

Reply:
Indeed, and also considered `.bin` or `.oct` as a nod to 8.3. I’m open to either design direction.